airflow/executors/kubernetes_executor.py

```diff
@@ -372,7 +374,8 @@ def __init__(self,
         self.log.debug("Kubernetes using namespace %s", self.namespace)
         self.kube_client = kube_client
         self.launcher = PodLauncher(kube_client=self.kube_client)
-        self.worker_configuration = WorkerConfiguration(kube_config=self.kube_config)
+        # Parse the airflow.cfg once on AirflowKubernetesScheduler initialization
```

Comment:
This comment doesn't make sense - airflow.cfg is already only parsed once at start-up

Reply:
You're correct. I meant call the `make_pod` method once, as opposed to the current setup where it is called each time we call `run_next`. Seeing as the majority of `WorkerConfiguration` is tied to the `airflow.cfg` there is no need to call `make_pod` each time `run_next` is called as it is completely static. This relates to your above comment.